lerna.json

```diff
@@ -1,4 +1,6 @@
 {
   "packages": ["packages/*"],
-  "version": "0.0.1"
+  "version": "0.0.1",
+  "npmClient": "yarn",
```

Comment:
yarn instead of npm ? I agree, but is it validated by the rest of the team ?

Reply:
It's a proposal :)

Reply:
For me it's ok too because NPm is not ready for resolutions.

Reply:
Agreed that's a pretty well working configuration managing lerna with yarn workspaces.
(npm 7 included some workspace notion but I've never played with it )